src/components/PersonLink.tsx

```diff
@@ -0,0 +1,70 @@
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { useContext } from 'react';
+import { Person } from '../types';
+import { PeopleContext } from '../PeopleContext';
+
+type Props = {
+  person: Person
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { personSlug } = useParams();
+  const {
+    people,
+  } = useContext(PeopleContext);
+
+  const findPersonFather = (newPeople: Person[], selectedPerson: Person) => {
+    const father = newPeople.find(p => p.name === selectedPerson.fatherName);
+
+    return father?.slug;
+  };
+
+  const findPersonMother = (newPeople: Person[], selectedPerson: Person) => {
+    const mother = newPeople.find(p => p.name === selectedPerson.motherName);
+
+    return mother?.slug;
+  };
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn('',
+        { 'has-background-warning': personSlug === person.slug })}
+    >
+      <td>
+        <a
+          href={`#/people/${person.slug}`}
+          className={cn('', { 'has-text-danger': person.sex === 'f' })}
+        >
+          {person.name}
+        </a>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      {findPersonMother(people, person)
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable